pyqtgraph/graphicsItems/ROI.py

```diff
@@ -1663,17 +1664,14 @@ def __init__(self, pos1, pos2, width, **args):
         d = pos2-pos1
         l = d.length()
         ang = Point(1, 0).angle(d)
-        ra = ang * np.pi / 180.
+        ra = radians(ang if ang is not None else 0.)
```

Comment:
In the future, maybe `Point` could use a way to ask for for the angle in radians.
Right now, Point calculates in radian, converts for return, only for the result to be converted back.